hera_mc/tests/test_correlator.py

```diff
@@ -1484,79 +1482,35 @@ def test_get_node_snap_from_serial_multiple_locs(mcsession):
     connection.start_gpstime = 1230375618
     mcsession.add(connection)
     mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple snap location numbers returned')
-    assert node == 701
-    assert snap_loc_num is None
+    pytest.raises(ValueError, mcsession._get_node_snap_from_serial, ['SNPD000703'])
```

Comment:
What error is this line testing? You should specify the text of the expected error not just that an error is raised. It looks like you took out the branch of the method that this was supposed to test. If you took it out because it's unreachable, we can just remove this test.

Reply:
How do you check a specified message?  Looks like ones uses `with pytest.raises`...?

Reply:
Here's an example from a pyuvdata test:
```    with pytest.raises(ValueError) as cm:
        uvdata_data.uv_object.check()
    assert str(cm.value).startswith('All antennas in ant_1_array must be in antenna_numbers')
```

Reply:
s=Sorry, formatting took a couple tries. The assert is **not** indented under the `with` statement.

Reply:
Deleted this error. check since new code doesn't allow it.